wg-releases/release-captain-overview.md

```diff
@@ -0,0 +1,65 @@
+# Release Captain
+
+This document is a guide to being a Release Captain in the Releases Working Group.
+
+## What is a release captain?
+
+Electron's stable release cycle mirrors the Chromium release cadence. Chrome currently releases on an 8 week cadence ([you can view Chromium's schedule here](https://chromiumdash.appspot.com/schedule)).
+
+As part of maintaining that release schedule, the Releases WG appoints a new "Release Captain" every 8 weeks. The Release Captain's primary role is to make sure each beta and stable release is released on schedule, and that all tasks supporting a given release are completed on time.
+
+The link for the Release Captain Schedule can be found on the [rotation website here](https://rotation.electronjs.org/rotations/d865ef86-ad57-4430-b417-e1d560bd8f93).
+
+## Rotation
+
+Each Release Captain Rotation lasts ~8 weeks. All current members of the Releases Working Group have been added to the Release Captain rotation.
+
+If you're not able to be the Release Captain during a specific period, or need others to cover your shift during specific weeks, post in the Releases WG Slack channel to find coverage.
+
+## Responsibilites
+
+### Ongoing (Week 1-8)
+* Review the current pre-release branch [project board] for new items
```

Comment:
nit: could all of these links be changed to be of the style `[project board][]` and `[meeting agenda][]`?

Would match what we enforce on `e/e`, and makes linting easier.